client/my-sites/site-settings/navigation.jsx

```diff
@@ -25,9 +25,8 @@ export default React.createClass( {
 			general: i18n.translate( 'General', { context: 'settings screen' } ),
 			writing: i18n.translate( 'Writing', { context: 'settings screen' } ),
 			discussion: i18n.translate( 'Discussion', { context: 'settings screen' } ),
-			analytics: i18n.translate( 'Analytics', { context: 'settings screen' } ),
+			traffic: i18n.translate( 'Traffic', { context: 'settings screen' } ),
```

Comment:
:information_source: This string already exists with the following **contexts**:
- `Help topic`

 Would it make sense to reuse one of the above?